src/topology.ts

```diff
@@ -164,7 +120,7 @@ const nodeEventHandler = (
     // Update snapshot
     snapshot.data[node].status = 'errored'
     snapshot.data[node].finished = new Date()
-    snapshot.data[node].error = error.toString()
+    snapshot.data[node].error = error.stack || error
```

Comment:
Why are we defaulting to `error.stack`? There are other properties on the error that are useful

Reply:
The error might have custom properties set on it. Ex:

```
let e = new ValidationError(...)
e.errors = [...]
```

Reply:
Done.